.github/workflows/continuous-integration.yml

```diff
@@ -343,7 +343,7 @@ jobs:
   release_artifacts:
     runs-on: ubuntu-22.04
     name: Release artifacts to OSS Sonatype
-    if: github.repository == 'kadai-io/kadai' && ( startsWith(github.ref, 'refs/tags') || github.ref == 'refs/heads/master' ) && github.head_ref == ''
+    if: github.repository == 'kadai-io/kadai' && startsWith(github.ref, 'refs/tags/v') && github.head_ref == ''
```

Comment:
Do we need to specify "/v"? Isn't it enough to be a tag?

Reply:
Probably not required. But it is a kind of QA to make sure, that all tags looks the same...